numpy/linalg/umath_linalg.c.src

```diff
@@ -1067,18 +975,20 @@ static NPY_INLINE void
             change_sign += (pivots[i] != (i+1));
         }
 
-        memcpy(sign,
-               (change_sign % 2)?
-                   &@cblas_type@_minus_one :
-                   &@cblas_type@_one
-               , sizeof(*sign));
+        if (change_sign % 2) {
+            *sign = @cblas_type@_minus_one;
+        }
+        else {
+            *sign = @cblas_type@_one;
+        }
```

Comment:
I worry that this might be been written this way to support unaligned pointers. @seberg, are gufunc inner loops guaranteed to have aligned data?

Reply:
Found this PR randomly. Yes, they are guaranteed to be aligned, so this is fine.

Might be nice to reactive the PR otherwise @ewmoore? It seems like an unfortunately cleanup to miss out on.